nix/release/default.nix

```diff
@@ -12,35 +12,28 @@ let
   # Version from the postgrest.cabal file (gotten with callCabal2nix).
   version = postgrest.version;
 
-  # Set of files that will be published in the GitHub release.
-  releaseFiles =
-    runCommand "postgrest-release-files"
```

Comment:
@monacoremo In case you're wondering. I deleted this nix function because I couldn't figure out how to pass a bash argument to it in `writeShellScriptBin "postgrest-release-github"`. My nix-fu isn't that good :smile:.